src/views/answerView.js

```diff
@@ -63,8 +66,19 @@ export const ShowRightAnswer = (answerListElement) => {
       answerElement.style.transform = "scale(1.1)"
       answerElement.style.color = "white"
       setTimeout(nextQuestion, 1500)
-  } i++
-}}
+    } i++ }
+  } else {
```

Comment:
Not sure why you need a separate else statement here, do you need to style the correct answer differently on the last question?

Reply:
I have to, to stop the skip button in the last page to execute the nextQuestion function, but also still giving me the right answer with style.

Reply:
OK I see. In that case there are two problems with the code, one is that you don't have i++ in the else branch, so you will not highlight the correct answer. The second is that you are repeating a lot of code here, so what I would do is separate the two parts of the code like so:

`let i = 0;
  for (const key of Object.keys(currentQuestion.answers)) {
    if (key === currentQuestion.correct) {
@@ -63,8 +66,19 @@ export const ShowRightAnswer = (answerListElement) => {
      answerElement.style.transform = "scale(1.1)"
      answerElement.style.color = "white"
      
  } 
i++;
}
if(quizData.currentQuestionIndex !== questions.length-1) {
  setTimeout(nextQuestion, 1500)
}`

So now we separate out the part that changes the style of the correct answer, and the code that moves to the next question, and it makes it a lot simpler.